### DISCUSSION TITLE
QTooltip does not clean up generated <div> tags when reused too quickly

### DISCUSSION THREAD

**Discussion by s0ft-decay** [quasarframework/quasar#9027] [General - Components / Directives / etc] [answered]
I made a fiddle of this: https://jsfiddle.net/t47ofw6k/

When you hover fast enough between the buttons to trigger the tooltip again before it has finished animating, the `<div>` tag created in the `<body>` is not removed. Otherwise, it works as intended, I'm just worried about the leaked `<div>` tags which can reach a very high count after a little while.

I'm using the tooltip incorrectly? I usually have a dedicated tooltip for each component, but I wanted to try the reusable tooltip this time around as per the documentation.

Thanks in advance.

**Comment by rstoenescu** [comment, marked as answer]
This is actually a bug. And just pushed a fix. Will be available in Quasar v2.0.0-beta.13.

**Comment by CrawfordW** [comment]
I have a similar problem - is it the same bug?...
I have a table with buttons on each row to delete that row. When over the button, it shows a tooltip. Sometimes after clicking the delete button, the tooltip remains visible even though the row is gone.
Also I have buttons on a drawer which launch something in the main window and retire the drawer. Sometimes the tooltip remains visible even though the button and drawer are gone.
